alacritty.yml

```
@@ -83,8 +83,7 @@
   # Specifying '0' will disable scrolling.
   #history: 10000
 
-  # Number of lines the viewport will move for every line scrolled when
-  # scrollback is enabled (history > 0).
+  # Modifier for multiplying the mouse scrolling distance.
```

```suggestion
  # Scrolling distance multiplier.
```

I've conferred with @kchibisov on IRC and we think just keeping it simple is best.

The `mouse` in my suggestion was incorrect, so it was no good since this obviously applies to the touchpad too.